.github/workflows/create-release-branch.yml

```diff
@@ -1,6 +1,17 @@
 name: create-release-branch
 on:
   workflow_dispatch:
+    inputs:
```

Comment:
However, I also thought it prudent to add in the manual option back in case we need to re-run (for whatever reason)

Reply:
Yeah, we definitely need this now.